### PR TITLE
containers project summary test

### PR DESCRIPTION
**Pull request by pavelzag**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 12b36d3edb5ec5be85d4ea6a16e371a6d5e68da8:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 00606749fff659209bafe05fb45af533bb5a1cc8:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/containers/test_containers_projects_summary.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+PROJECTS_PROPERTIES_FIELDS = ['Name', 'Creation timestamp', 'Resource version']
+PROJECTS_RELATIONSHIPS_FIELDS = ['Containers Provider', 'Routes', 'Services', 'Replicators',
+                                 'Pods', 'Nodes']
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_projects_summary.py]: Locator in test file, please use one from model. Ask Tony.

File: cfme/tests/containers/test_containers_projects_summary.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        """
+    sel.force_navigate('containers_projects')
+    project_name = [r.name.text for r in list_tbl.rows()]
+    for name in project_name:
+        obj = Project(name, provider)
+        val = obj.get_detail('Relationships', ''.join(projects_relationships_fields))
+        assert val
+
+
+@pytest.mark.parametrize('projects_properties_fields', product(PROJECTS_PROPERTIES_FIELDS))
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_projects_summary.py]: Do not use product on a single list

**Comment by psav** [comment]
Some of this seems to be included in another PR, can this be closed?

**Comment by pavelzag** [comment]
Yes, please close if you see duplication

**Pull request closed without merging**